storage/tsdb/tsdb.go

```diff
@@ -58,11 +58,11 @@ type Options struct {
 func Open(path string, r prometheus.Registerer, opts *Options) (*tsdb.DB, error) {
 	// Start with smallest block duration and create exponential buckets until the exceed the
 	// configured maximum block duration.
-	rngs := tsdb.ExponentialBlockRanges(int64(time.Duration(opts.MinBlockDuration).Seconds()*1000), 3, 10)
+	rngs := tsdb.ExponentialBlockRanges(int64(time.Duration(opts.MinBlockDuration).Seconds()*1000), 10, 3)
 
-	for i, v := range rngs {
+	for i, v := range rngs[1:] {
```

Comment:
I'm not familiar with the API used here, but should you add a bounds check to prevent a potential panic if a slice of length 0 or 1 is returned?